### PR TITLE
add apps.py and translation file

### PR DESCRIPTION
**Pull request by 9nix00**
thanks django-taggit, it saves my time!

this is my contribute. hope this is useful. please check it.

### CODE REVIEW

**Comment by prydie** [comment]
CI is failing due to icorrectly ordered imports in `taggit/apps.py`. If you could fix this so that the rest of the test suite runs that would be great.

```
ERROR: /home/travis/build/alex/django-taggit/taggit/apps.py Imports are incorrectly sorted.
```

**Comment by 9nix00** [comment]
upgrade,please confirm it.

File: taggit/apps.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig as BaseConfig
+from django.utils.translation import ugettext_lazy as _
+
+class AppConfig(BaseConfig):
```
Comment (prydie) [inline taggit/apps.py]: The standard around App config is to name the config class based on the module name. For example:

 ```python
 # rock_n_roll/apps.py

from django.apps import AppConfig

class RockNRollConfig(AppConfig):
    name = 'rock_n_roll'
    verbose_name = "Rock ’n’ roll"
```
From the [docs](https://docs.djangoproject.com/en/1.9/ref/applications/#for-application-authors).

**Comment by prydie** [comment]
One suggestion code wise. Not being a Chinese speaker though I can't review the translation, unfortunately.

File: taggit/apps.py (hunk 1 above)
Comment (frewsxcv) [inline taggit/apps.py]: In other words, just rename this to `TaggitAppConfig` and this should be good to merge

**Comment by frewsxcv** [comment]
Looks great, thanks!

**Pull request merged**